### ISSUE TITLE
[A11Y] Dropdown toggle buttons missing accessibility label

### ISSUE THREAD

**Issue by davwheat** [flarum/core#2581]
## Bug Report

**Current Behavior**
All dropdown toggle buttons (the ones with only carets) are missing accessible labels.

![image](https://user-images.githubusercontent.com/7406822/106191901-62cb8280-61a3-11eb-9764-0cc39336ca70.png)

**Expected Behavior**
Accessibility labels should be provided on all interactive elements with purely visual explanations, like these. Preferably something similar to `aria-label="Show more options"`, with the text being part of language packs.

Without accessible labels, screen-readers will only hear "Button".

**Environment**
- Flarum version: 0.1.0-beta.15

**Comment by SychO9** [comment]
Thank you for reporting this!
